internal/providers/aws/aws.go

```diff
@@ -58,15 +59,15 @@ func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
 }
 
 func NewFetcher(l *log.Logger) (resource.Fetcher, error) {
-	sess, err := session.NewSession(&aws.Config{})
+	cfg, err := config.LoadDefaultConfig(context.Background())
 	if err != nil {
 		return resource.Fetcher{}, err
 	}
-	sess.Config.Credentials = ec2rolecreds.NewCredentials(sess)
+	cfg.Credentials = aws.NewCredentialsCache(ec2rolecreds.New())
```

Comment:
this line can be dropped. LoadDefaultConfig when run on an EC2 instance will use the EC2 creds

Reply:
cool! removed line and updated vendoring